components/renku_data_services/migrations/versions/788840ccdcf4_add_connected_services.py

```diff
@@ -30,7 +30,7 @@ def upgrade() -> None:
         sa.Column("kind", sa.Enum("gitlab", "github", name="providerkind"), nullable=False),
         sa.Column("scope", sa.String(), nullable=False),
         sa.Column("url", sa.String(), nullable=False),
-        sa.Column("client_secret", sa.String(length=500), nullable=True),
+        sa.Column('client_secret', sa.LargeBinary(), nullable=True),
```

Comment:
formatting error? This should be double quotes.

Reply:
Done in 73ba30f9b838edb452ca4fb3d9a6e12bc2f42299.